spine-c/src/spine/Bone.c

```diff
@@ -209,9 +206,9 @@ float spBone_getWorldScaleY (spBone* self) {
 }
 
 void spBone_worldToLocal (spBone* self, float worldX, float worldY, float* localX, float* localY) {
-	float x = worldX - self->worldX, y = worldY - self->worldY;
 	float a = self->a, b = self->b, c = self->c, d = self->d;
 	float invDet = 1 / (a * d - b * c);
+	float x = worldX - self->worldX, y = worldY - self->worldY;
 	*localX = (x * d * invDet - y * b * invDet);
 	*localY = (y * a * invDet - x * c * invDet);
 }
```

Comment:
Maybe I misunderstood your comment. I assume that it is about the "usefulness" of this change :smile: I did almost all the changes (even cosmetic ones) to make it simpler to find the differences between runtimes easier in the future :-)